cli/verify-translations.js

```diff
@@ -0,0 +1,51 @@
+const chalk = require('chalk'),
+	fs = require('fs'),
+	path = require('path');
+
+const langs = ['ar', 'da', 'de', 'es', 'fr', 'ja', 'ko', 'nl', 'pt', 'sv', 'tr', 'zh-tw', 'zh'];
+const langTermsPath = path.join(__dirname, '../lang');
+
+function _parseFile(fileName) {
+	const file = fs.readFileSync(`${langTermsPath}/${fileName}.js`).toString();
+	const stringContent = file.split('default ')[1].split(';')[0];
```

Comment:
This relies on the only semi-colon being at the end of the file. Is this too risky and I should split on something like `};` and then add the `}`?

Reply:
Seems OK for now. NPM is pretty close to supporting ES modules natively, although it may require that the extension be `.mjs` unfortunately.